components/rating/rating.html

```diff
@@ -0,0 +1,102 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="rating.css">
+    <link rel="stylesheet" href="../component.css">
+    <link rel="stylesheet" href="../main.css">
+</head>
+
+<body>
+    <div class="navigation-bar">
+        <div class="logo">
+            <a href="../../index.html">
+                <i class="fa fa-eject logo-img" aria-hidden="true"></i>
+                <h3>BoostMat UI</h3>
+            </a>
+
+        </div>
+        <div class="humber-icon">
+            <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+        </div>
+
+    </div>
+
+    <div class="container">
+        <div class="sidebar-wrapper trans-off">
+            <div class="sidebar">
+                <div class="sidebar-header">
+                    <div class="logo">
+                        <a href="../../index.html">
+                            <i class="fa fa-eject logo-img" aria-hidden="true"></i>
+                            <h3>BoostMat UI</h3>
+                        </a>
+                    </div>
+                </div>
+                <a href="../main.html" class="install-text">Install BoostMat UI</a>
+                <div class="sidebar-main">
+                    <a href="../alert/alert.html">Alert</a>
+                    <a href="../avatar/avatar.html">Avatar</a>
+                    <a href="../badge/badge.html">Badge</a>
+                    <a href="../button/button.html">Button</a>
+                    <a href="../card/card.html">Card</a>
+                    <a href="../image/image.html">Image</a>
+                    <a href="../input/input.html">Input</a>
+                    <a href="../lists/lists.html">Lists</a>
+                    <a href="../modal/modal.html">Modal</a>
+                    <a href="../navigation/navigation.html">Navigation</a>
+                    <a class="active-page" href="../rating/rating.html">Rating </a>
+                    <a href="../grid/grid.html">Responsive Grid</a>
+                    <a href="../text-utilities/textUtilities.html">Text Utilities</a>
+                    <a href="../toast/toast.html">Toast</a>
+                </div>
+                <div class="icon-bar">
+                    <a href="https://github.com/rutvikpumak" class="github-logo"><i class="fa fa-github"></i></a>
+                    <a href="https://twitter.com/rutvikumak13" class="twitter"><i class="fa fa-twitter"></i></a>
+                    <a href="https://linkedin.com/in/rutvikumak" class="linkedin"><i class="fa fa-linkedin"></i></a>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="section">
+
+            <div class="section-header">
+                <div>
+                    <h1 class="section-main-text-header">Ratings</h1>
+                    <p class="paragraph">
+                        Rating component is used to give start rating for a product / order.
```

Comment:
Just a little typo --> give Star* rating

Reply:
Thank you